### ISSUE TITLE
unable to save only the comment

### ISSUE THREAD

**Issue by dpaakash** [dpaakash/food-diary#15]
comment without food entries is not being saved

**Comment by dpaakash** [comment]
```
SELECT public.food_items.item_name, public.date_entry.comment from public.food_entries
    JOIN public.food_items ON public.food_entries.item_id = public.food_items.item_id 
    JOIN public.date_entry ON public.food_entries.date_id = public.date_entry.date_id
    WHERE date=to_date('Wed Oct 02 2019','Dy Mon dd yyyy')
```

need left join

**Comment by dpaakash** [comment]
fixed in d85469bd78759aaa8e151252f4d9dac236ca5a52
